CrossClusterDataSteps.md

```diff
@@ -99,7 +99,7 @@ oc process -f ./cross-cluster/target-migrator-tmpl.yaml \
 
 4. Get token for remote secret value and create secret manifest for source (shortcut below)
 ``` bash
-MIG_TOKEN_SECRET=`oc get secret | grep target-pvc-migrator-token | head -1 | awk '{print $1}'`
+MIG_TOKEN_SECRET=`oc get secret -o json | jq '.items[] | select(.type == "kubernetes.io/service-account-token") | .metadata.name' | grep target-pvc-migrator-token | head -1 | awk '{print $1}'`
```

Comment:
What do you think of: 
```suggestion
MIG_TOKEN_SECRET=`oc get secret -o json | jq '.items[] | select((.type == "kubernetes.io/service-account-token") and (.metadata.name | contains("target-pvc-migrator-token"))) | .metadata.name' | head -1`
```

Almost there.. couldn't get firstmatch working, so had to keep the `head -1` in the command string, but able to remove the grep and the awk commands.